composer/models/huggingface.py

```diff
@@ -23,22 +25,29 @@
 from composer.models.base import ComposerModel
 from composer.utils import MissingConditionalImportError, dist, get_file, import_object, is_model_fsdp, safe_torch_load
 
+try:
+    from peft import PeftModel, get_peft_model
+    peft_installed = True
```

Comment:
nit: perhaps `is_peft_installed`

Reply:
Originally I had `_is_peft_installed`, but turns out everywhere else in composer we do it like `library_installed`, so I'm keeping this.